setup.py

```diff
@@ -19,7 +19,7 @@ def read(filename):
         'boto': ['boto>=2.32.0'],
         'boto3': ['boto3>=1.4.4'],
         'dropbox': ['dropbox>=7.2.1'],
-        'google': ['google-cloud-storage>=0.22.0'],
+        'google': ['google-cloud-storage>=1.15.0'],
```

Comment:
Is this change required?

Reply:
The code in this pull request relies on the fact that `Blob.generate_signed_url` has a `api_access_endpoint` parameter (introduced in https://github.com/googleapis/google-cloud-python/pull/7460 and google-cloud-storage 1.15.0).

It would be theoretically possible to update the code so that it is still compatible with google-cloud-storage 0.22.0, but because of the substantial differences in google-cloud-storage pre- and post-1.15.0, this would take a substantial amount of work and would significantly increase the complexity of this pull request's code.

Reply:
That change was merged in April. I'm not sure if I can unilaterally require that upgrade.

Is the major difference that non-public URLs wouldn't be able to be signed?